NSS-Day-Cohort-42/rare-API-media-moguls
# User fixtures added.

#### Changes Made
1. Added file `tokens.json`, `users.json`, `rareusers.json` to `fixtures` directory.
1. Added linter exception file 

#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout mp-user-fixtures
```
2. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
